Configuration/Eras/python/Modifier_run3_ecal_devel_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+run3_ecal_devel =  cms.Modifier()
```

Comment:
I'm not sure about this modifier... First, `_devel` seems very general it could be any development. Then besides this cosmetic comment, the only thing it does that it changes the output collections. This seems like something you can do with a cmsDriver command and should not require a specific era modifier.

Reply:
I chose the `_devel` suffix basically to be consistent with `Configuration/Eras/python/Modifier_phase2_ecal_devel_cff.py`. That one is also for generic phase-II development, and I think this one could be used in the future for generic Run-3 development. When I tried, it was really difficult to get the ecalTimeDigitizer to run without doing it in a similar way to how it does it for phase 2 ([here](https://github.com/cms-sw/cmssw/blob/ac6fa58e8d78cd0dd43993bf00ce753263e3aa49/SimGeneral/MixingModule/python/digitizers_cfi.py#L75-L77)).

Reply:
ok, let's just keep this as is then, thanks